swagger.yaml

```diff
@@ -2257,6 +2257,16 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
+  operators:
```

Comment:
To make it easier to add operators in future, this could be done similar to managed-domains:

list-operators:
    type: array
    items:
      $ref: '#/definitions/operator'

 operator:
    type: object
    properties:
        enabled:
           type: boolean
           default: false

Reply:
The thought was that OCS would have more properties in the future which would be configurable via the ocs object.  For example, storage type (block/file/object), encryption enabled, cores/disk, etc.